config/schema/metatag_context.schema.yml

```diff
@@ -0,0 +1,32 @@
+metatag.metatag_context.*:
+  type: config_entity
+  label: 'Metatag'
+  mapping:
+    id:
+      type: string
+      label: 'ID'
+    label:
+      type: label
+      label: 'Type'
+    uuid:
+      type: string
+    title:
+      type: string
+      label: 'Page title'
+    description:
+      type: string
+      label: 'Description'
+    abstract:
+      type: string
+      label: 'Abstract'
+    keywords:
```

Comment:
Agree. I think that we could provide such flexibility with a tags field of type sequence. The sequence would contain a set of Tag config entities. Does this make sense to you? I described this with more details at https://paper.dropbox.com/doc/Drupal-8s-Metatag-E8j9mLHoJtojhaIssmVEF#:h=Proposed-architecture-to-manag.